src/Router.php

```diff
@@ -105,13 +105,27 @@ public function setOptions(array $options)
 
         if (!empty($options['routes'])) {
             foreach ($options['routes'] as $path => $routeConfig) {
-                $defaults = isset($routeConfig['defaults']) ? $routeConfig['defaults'] : [];
-                $requirements = isset($routeConfig['requirements']) ? $routeConfig['requirements'] : [];
-                $this->addRoute(new Route($path, $defaults, $requirements));
+                $name = $path;
```

Comment:
The other way around would make a bit more sense here:

`... as $name => $routeConfig) {`

and then:
```
if (array_key_exists('path', $routeConfig)) {
    $path = $routeConfig['path'];
} else {
    $route = $name;
}
```